tests/gccgo/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check that snapd builds with gccgo
+prepare: |
+    echo Installing gccgo-6 and pretending it is the default go
+    sudo apt install -y gccgo-6
+    sudo ln -s /usr/bin/go-6 /usr/local/bin/go
+restore: |
+    rm -f /usr/local/bin/go
+    sudo apt-get autoremove -y gccgo-6
+execute: |
+    echo Build with gccgo with tests
+    pwd
+    sudo -i -u test /bin/sh -c "cd /gopath/src/github.com/snapcore/snapd && dpkg-buildpackage -Zgzip"
```

Comment:
We can also do it separately of course. This was just the easiest way to simulate what a buildd is doing when building the package on an architecture that does not have native go (like powerpc).